starfish/viz.py

```diff
@@ -1,10 +1,10 @@
 import numpy as np
-from showit import image, tile
 
 from .stats import im_describe, stack_describe
 
 
 def tile_lims(stack, num_std, bar=True, size=20):
```

Comment:
are these functions still used?

Reply:
They might still be in some notebooks. I think they deserve some modernization, but probably shouldn't be removed yet.